src/BusinessLogic/User/UserAccountService.php

```diff
@@ -104,7 +97,7 @@ public function login($apiKey)
      */
     public function setDefaultParcel($force)
     {
-        $parcelInfo = $this->configuration->getDefaultParcel();
+        $parcelInfo = $this->getConfigService()->getDefaultParcel();
         if ($parcelInfo === null || $force) {
             if ($this->setParcelInfoInternal()) {
```

Comment:
Do you think the name of the function is a bit confusing? it says `set...` but is not setting anything

Reply:
If you are referring to the `setParcelInfoInternal`, it is used as an extension point in modules. If the module has set the parcel from the shop, we don't need to continue setting it from Packlink PRO. If not, we fetch the parcel info from Packlink API and set it. In both cases, this method is setting the default parcel information from whatever source it can fetch it.